integration/scripts/src/runTest.ts

```diff
@@ -0,0 +1,76 @@
+import { ChainlinkNode } from './chainlinkNode'
+import { fetchTests } from './tests'
+import { fetchArgs, fetchCredentials } from './common'
+import * as assert from './asserts'
+
+const main = async () => {
+  const tests = fetchTests()
+
+  const { chainlinkUrl } = fetchArgs()
+  const credentials = fetchCredentials()
+  const node = new ChainlinkNode(chainlinkUrl, credentials)
+
+  const ctx = await assert.context(async (ctx) => {
+    for (let i = 0; i < tests.length; i++) {
+      const test = tests[i]
+      await assert.newTest(test, async () => {
+        const jobCount = (await node.getJobs()).meta?.count || 0
```

Comment:
Can we take `jobCount` out from the loop? Doesn't seem that it changes

Reply:
Job count *should* change every loop after the job has been created.

We're checking the job count before creating the job, and after, and we're asserting that the new job count is the old one + 1.